lib/src/base-styles/_nx-tile.scss

```diff
@@ -157,8 +157,21 @@ $nx-tile-background-color: $nx-white;
 
   border-top: 1px solid #e9e9e9;
   margin-top: $nx-spacing-md;
-  padding: $nx-spacing-xl 0;
+  padding: $nx-spacing-l 0;
   text-align: right;
+
+  .nx-btn-bar {
+    margin: 0;
+    text-align: right;
```

Comment:
My understanding is that _all_ `.nx-btn-bar`s should right-align their contents, not just ones in tile footers (or form footers).

Reply:
> My understanding is that _all_ `.nx-btn-bar`s should right-align their contents, not just ones in tile footers (or form footers).

I changed this in the form layout PR